db/migrate/20200515161631_create_payment_card.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePaymentCard < ActiveRecord::Migration[6.0]
+  def change
+    create_table :payment_cards do |t|
+      t.timestamps
+
+      t.string :quote_number
+      t.string :proxy_number
+      t.string :card_number
+      t.string :expiration_date
+      t.string :cvv_number
```

Comment:
Add

- Sequence Number
- Client Order Number

Remove
- Expiration Date
- CVV Number

Reply:
Also, let's add:

- Activation Code
- Activated At

I think we may end up duplicating some fields on the Aid Application, but I think that'll be fine.